app/views/post_office/pending_service.text.erb

```diff
@@ -5,7 +5,7 @@ A new service is pending for review on the admin page.
 Name: <%= h @service.name %>
 Account: <%= h @service.account.org_name %>
 
-<%= link_to 'View list of pending services', admin_services_path %>
+<%= link_to 'View the pending service', admin_service_path(@service) %>
```

Comment:
shouldn't this stay plural and link to the dashboard?

Reply:
@thomasmaas I don't know, this one is the one I left in a different commit intentionally because I was doubting the same that you are asking now... 😄

Reply:
I don't know how a service can be pending…is this email sent to provider? master?

Reply:
Ok, so Service used to have the pending state but not anymore 😄 So everything related to this state should probably be removed... 🤔But this also happens in master so I wouldn't take the risk for now yet 😄